source/ocsp-support/ocsp-support.rst

```diff
@@ -156,6 +156,16 @@ per MongoClient basis.
    ``tlsAllowInvalidCertificates=true`` are specified alongside the
    option to enable certificate revocation checking.
 
+TLS Requirements
+----------------
+`Server Name Indication
+<https://en.wikipedia.org/wiki/Server_Name_Indication>`__ (SNI) MUST BE
+used in the TLS connection that obtains the server's certificate,
```

Comment:
Realistically, I believe only an issue with Atlas free tier's proxy, as suggested by @divjotarora.

Reply:
It may also be necessary for shared Atlas tiers? Not sure how those work.

Reply:
That's how I assume it would affect us.